charts/platform/config/otel-config.yaml

```diff
@@ -0,0 +1,68 @@
+receivers:
+  otlp:
+    protocols:
+      http:
+        endpoint: :4318
+processors:
+  transform:
+    #    error_mode: ignore
+    trace_statements:
+      - context: spanevent
+        conditions:
+          - (name == "sync.finish.stats" and attributes["sync.resources"] == 0 and attributes["sync.errors"] == 0 and attributes["sync.panics"] == 0)
+        statements:
+          - set(span.attributes["cq.delete"], 1)
+  #      - context: span
+  #        statements:
+  #          - delete_key(attributes, "sync.client.id")
+
+  filter:
+    #    error_mode: ignore
+    traces:
+      span:
+        - attributes["cq.delete"] == 1
+    logs:
+      log_record:
+        - 'IsMatch(body, "table resolver finished with error|table sync finished|table resolver finished with panic") == false'
+
+  batch:
+    send_batch_size: 100000
+    timeout: 5s
+
+exporters:
+  nop:
+  clickhouse:
+    endpoint: ${env:CLICKHOUSE_DSN}
+    # database: ${env:CLICKHOUSE_DATABASE}
```

Comment:
This wasn't needed? because the [example has it](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/clickhouseexporter/example/otel-collector-config.yml#L27-L28)

I think we're lucky that it defaults to `default` in the source [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/clickhouseexporter/config.go#L85).

Reply:
Or I think it's not needed if the DSN contains a path:
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/clickhouseexporter/config.go#L151-L153